perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -53,6 +53,27 @@ sub users_can_hide { return 1; }
 
 sub default_show_name { 0 }
 
+sub lookup_by_ref_regex {
+    return qr/^\s*((?:ENQ)?\d+)\s*$/;
+}
+
+sub lookup_by_ref {
+    my ($self, $ref) = @_;
+
+    my $criteria = [
```

Comment:
Don't need this now due to your other change; you can return the criteria if `$ref =~ /^ENQ/` and nothing if not and it'll fall back.